webpack.config.js

```diff
@@ -42,6 +42,10 @@ module.exports = {
       templateParameters: compilation => ({
         jsSource: `\${createLinkTo(dir:'/js', file:'bundle.${compilation.hash}.js')}`,
         cssSource: `\${createLinkTo(dir:'css/', file:'bundle.${compilation.hash}.css')}`,
+        // eslint-disable-next-line no-template-curly-in-string
+        gaEnabled: '${grailsApplication.config.google.analytics.enabled}',
+        // eslint-disable-next-line no-template-curly-in-string
+        webPropertyID: '${grailsApplication.config.google.analytics.webPropertyID}',
```

Comment:
Oh shit we do actually do this at build time. :confused:

Reply:
That scares me a bit since someone would conceivably want to change the config properties at runtime. In fact, this is a bigger issue and they would try to change them, but changing them would have no effect.